Source/WebKit/DerivedSources.make

```diff
@@ -404,24 +406,28 @@ all : $(SANDBOX_PROFILES_WITHOUT_WEBPUSHD) $(WEBPUSHD_SANDBOX_PROFILE) $(SANDBOX
 %.sb : %.sb.in
 	@echo Pre-processing $* sandbox profile...
 	grep -o '^[^;]*' $< | $(CC) $(SANITIZE_FLAGS) $(SDK_FLAGS) $(TARGET_TRIPLE_FLAGS) $(SANDBOX_DEFINES) $(TEXT_PREPROCESSOR_FLAGS) $(FRAMEWORK_FLAGS) $(HEADER_FLAGS) $(EXTERNAL_FLAGS) -include "wtf/Platform.h" - > $@
-	if [ -e "/usr/local/bin/sbutil" ]; then \
-		if [[ $(SDK_NAME) =~ "iphoneos" ]]; then \
+	xcrun --sdk $(SDK_NAME) -f sbutil; \
+	if [[ $$? == 0 ]]; then \
+		if [[ $(SDK_NAME) =~ "iphone" || $(SDK_NAME) =~ "watch" || $(SDK_NAME) =~ "appletv" ]]; then \
 			if [[ $* == "com.apple.WebKit.adattributiond" || $* == "com.apple.WebKit.webpushd" ]]; then \
-				sbutil compile -D IMPORT_DIR=$(SDKROOT)/usr/local/share/sandbox/profiles/embedded/imports $@ > /dev/null; \
+				if [ ! -e $(SANDBOX_IMPORT_DIR) ]; then \
+					exit 0; \
+				fi; \
```

Comment:
Is it expected that some configurations may have `sbutil` but not SANDBOX_IMPORT_DIR? I would have guessed that what you're really checking for is whether we're on an internal SDK, which you could do by checking if `USE_INTERNAL_SDK` is YES.

Reply:
Yes, some configurations have sbutil, but not SANDBOX_IMPORT_DIR.

Thanks for reviewing!